frontend/.eslintrc

```diff
@@ -1,11 +1,40 @@
 {
-	"extends": ["@isnotdefined/eslint-config"],
-	"rules": {
-		"@typescript-eslint/no-require-imports": "warn",
-		"jsdoc/require-jsdoc": "off",
-		"jsdoc/require-description": "off"
-	},
-	"parserOptions": {
-		"project": "./tsconfig.json"
-	}
+  "root": true,
```

Comment:
For me it would take to long to go through that whole file... just use the example I gave you from ngx-crud-playground

Reply:
https://github.com/henryruhs/ngx-crud-playground/blob/master/.eslintrc